app/services/text_handler.rb

```diff
@@ -1,48 +1,61 @@
 class TextHandler
 
-  def process_input(user, params)
-    body = params[:Body].upcase
-
-    case body
-    when "Y"
-      if user.confirmed_at.blank?
-        user.confirmed_at = DateTime.now
-        user.confirmation_sent_at = DateTime.now
-        user.save
-        I18n.t('texts.confirmation_success_response')
+  def process_input(user, input, is_text)
```

Comment:
Can we add some comments about what each of these are doing before the conditionals, just for some clarity?